services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/entity/VpcResourceMeta.java

```diff
@@ -0,0 +1,25 @@
+package com.futurewei.alcor.netwconfigmanager.entity;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class VpcResourceMeta {
+
+    private String vni;
+
+    // Private IP => List<ResourceMetadata>
+    private HashMap<String, List<ResourceMeta>> resourceMetas;
```

Comment:
@xieus  What's the IP here?

Reply:
Here is the private ip within the VPC ip space.